#! /bin/sh /usr/share/dpatch/dpatch-run
## 02-recordings-refresh.dpatch
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Patch for the Recordings menu of the Remote Timers plugin
## DP: reload the recordings list each time the menu is accessed

@DPATCH@
Index: menu.c
===================================================================
--- a/menu.c	(revision 629)
+++ b/menu.c	(working copy)
@@ -1835,6 +1835,7 @@
   SetFreeDiskDisplay(true);
   OpenSubMenus |= replayEnded;
   replayEnded = false;
+  Recordings.Update(true);
   if (Current() < 0)
      SetCurrent(First());
   else if (OpenSubMenus && cReplayControl::LastReplayed() && Open(true))
